src/sass_types/factory.cpp

```diff
@@ -45,26 +43,27 @@ namespace SassTypes
     }
   }
 
-  void Factory::initExports(Handle<Object> exports) {
-    Local<Object> types = NanNew<Object>();
-    exports->Set(NanNew("types"), types);
+  void Factory::initExports(v8::Local<v8::Object> exports) {
+    Nan::EscapableHandleScope scope;
```

Comment:
oh, and this could also use `NAN_MODULE_INIT` if it's being fed from `NODE_MODULE`.